MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
+------------------
+|   @achow101  | wallet, PSBTs |
+|   @fanquake | general, tooling, build system |
+|   @hebasto | UI/UX |
+|   @laanwj | general |
+|   @marcofalke | general |
+|   @sipa | general, peer-to-peer, consensus, cryptography |
+
+See [REVIEWERS](/REVIEWERS) for more information on specific focus areas of
+maintainers and regular contributors.
+
+
+## Regular Contributors, Scoped Maintainers, General Maintainers, and Lead Maintainers
+
+The title of Maintainer in Bitcoin is more of a "janitorial responsibility"
+than a position of privilege. However, the role does include permissions on the
+repository, and connotes a general sense of stewardship over the project.
+
+
+Largely, the role of maintainers is to decide to merge Pull Requests. This
+process is described in [CONTRIBUTING.md](/CONTRIBUTING.md), but as an excerpt:
+
+> Whether a pull request is merged into Bitcoin Core rests with the project merge
```

Comment:
Ok -- I can move the section, that just felt a bit more invasive because that section is generally useful to have in contributors, but maybe I can change to a link from CONTRIBUTORS.md to MAINTAINERS.md?

Reply:
Hi there text me here jacksonrose799@gmail.com